moveit_core/collision_detection/src/collision_tools.cpp

```diff
@@ -39,41 +39,44 @@
 
 namespace collision_detection
 {
-void getCostMarkers(visualization_msgs::MarkerArray& arr, const std::string& frame_id,
+void getCostMarkers(visualization_msgs::msg::MarkerArray& arr, const std::string& frame_id,
                     std::set<CostSource>& cost_sources)
 {
-  std_msgs::ColorRGBA color;
+  std_msgs::msg::ColorRGBA color;
   color.r = 1.0f;
   color.g = 0.5f;
   color.b = 0.0f;
   color.a = 0.4f;
-  getCostMarkers(arr, frame_id, cost_sources, color, ros::Duration(60.0));
+  getCostMarkers(arr, frame_id, cost_sources, color, rclcpp::Duration(60.0));
```

Comment:
```suggestion
  getCostMarkers(arr, frame_id, cost_sources, color, rclcpp::Duration(60, 0));
```
`rclcpp::Duration` doesn't support the seconds constructor anymore.

Reply:
done